WinUIGallery/Helper/TitlebarHelper.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using AppUIBasics.Helper;
+using Windows.Storage;
+using Windows.Storage.Pickers;
+using Windows.System;
+using Microsoft.UI;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+using Microsoft.UI.Xaml.Navigation;
+using WinRT;
+using System.Runtime.InteropServices;
+
+namespace WinUIGallery.DesktopWap.Helper
+{
+    internal class TitlebarHelper
+    {
+
+        [ComImport, Guid("EECDBF0E-BAE9-4CB6-A68E-9598E1CB57BB"), InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
```

Comment:
You don't need this ComImport. You can use WinRT.Interop.WindowNative.GetWindowHandle(window) to get the hwnd.

Reply:
Thanks for the tip @marb2000! I updated the code.